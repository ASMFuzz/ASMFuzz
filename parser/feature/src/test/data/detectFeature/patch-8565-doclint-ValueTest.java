@@ -65,4 +65,16 @@ public enum E {
 
     /** invalid enum constant: {@value Thread.State#NEW} */
     public int badEnum;
+
+    /** valid: {@value %04x} */
+    public static final int maxShort = 65535;
+
+    /** valid: {@value "%5.2f"} */
+    public static final double pi = 3.14159265358979323846;
+
+    /** invalid format: {@value %%04x} */
+    public static final int f3 = 0;
+
+    /** invalid format: {@value "04x"} */
+    public static final int f4 = 0;
 }