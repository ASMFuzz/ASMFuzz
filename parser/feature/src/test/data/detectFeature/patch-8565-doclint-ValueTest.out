@@ -19,4 +19,10 @@ ValueTest.java:63: error: value does not refer to a constant
 ValueTest.java:66: error: value does not refer to a constant
     /** invalid enum constant: {@value Thread.State#NEW} */
                                ^
-7 errors
+ValueTest.java:75: error: invalid format: %%04x
+    /** invalid format: {@value %%04x} */
+                                ^
+ValueTest.java:78: error: invalid format: "04x"
+    /** invalid format: {@value "04x"} */
+                                ^
+9 errors